### ISSUE TITLE
Obfuscated python scripts include object names, string etc. in clear text.

### ISSUE THREAD

**Issue by jondy** [dashingsoft/pyarmor#5]
After encrypting a module it doesn't seem to be encrypted. Looking at the binary source, I still find all object names, strings etc. in clear text. Is that the intended behaviour?

**Comment by jondy** [comment]
In pyarmor 3.2.0, only co_code is obfuscated, co_names and co_const is separated saved in .pyc file. So they're clear text.

Next released version 3.2.1, most of constant will be obfuscated, for example,

def hello(arg="first"):
  name = "jondy"
  users = [ "bob", "time", 3]

In this function, all the constant string: "first", "jondy", "bob", "time" will be obfuscated.
co_names, like "hello", "arg", "name", "users" will still in literal

**Comment by jondy** [comment]
In the minor version Pyarmro 3.3.0, there are 2 new modes: 7, 8. The default mode is 8.

* mode 7: only code object of python module is obfuscated, it's almost
  quickly as no obfuscated module.
  
```
    python pyarmor.py --with-capsule=project.zip --output=dist --mode=7 foo.py
    
```

* mode 8: both code object and bytecode are obfuscated, it's slower
  but more secure. It's the default mode.

```
    python pyarmor.py --with-capsule=project.zip --output=dist --mode=8 foo.py

```

Both of modes generate obfuscated scripts as the following:

```
__pyarmor__(__name__, b'xxxx')
```
